disk_areas.md

```diff
@@ -30,7 +30,9 @@ Abel Disk areas that are relevant for your work at the Veterinary Institute
 
 5. **$SCRATCH**: when a job is started, a **temporary area** on $SCRATCH is automatically created. It is also automatically deleted when the job finishes. This allows the jobs to run faster without interfering with the work of others. You can access $SCRATCH for status monitoring of running jobs: with `/work/jobs/JOBID.d` where JOBID.d is the ID that has been provided when you submitted the job in the queue system SLURM (thus accessible only during job's lifetime).
 
-6. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`. You should put files required for analysis here and run analysis from this area (via SLRUM script or other mechanisms).  
+6. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`. You should put files required for analysis here and run analysis from this area (via SLRUM script or other mechanisms).
```

Comment:
May I suggest that you begin with saying that while Bifrost puts files there itself, for other jobs have to put the files there themselves? That way we begin with how they get there, and then we say how they are deleted.

Reply:
ok -done